packages/docs/src/examples/va-divider/Inset.vue

```diff
@@ -2,12 +2,16 @@
   <div class="my-5 flex lg4">
     <p class="example-item">
       <va-icon name="home" />
-      <span class="float-right">I'm a list item</span>
```

Comment:
Most likely you can keep this text as generic.

Reply:
did the same